### ISSUE TITLE
refactor use-any can break existing code

### ISSUE THREAD

**Issue by lonico** [sourcery-ai/sourcery#191]
### Issue description or question

```
-        self.ontap_version['valid'] = True
-        for key in self.ontap_version:
-            if self.ontap_version[key] == -1:
-                self.ontap_version['valid'] = False
-                break
+        self.ontap_version['valid'] = all(
+            self.ontap_version[key] != -1 for key in self.ontap_version
+        )
```

(variable) ontap_version: dict[str, str | int]

Sourcery - Use any() instead of for loop, Invert any/all to simplify comparisons sourcery(refactoring)

The proposed change fails when self.ontap_version['valid'] == -1.

(I agree the existing code is icky.  A value of None or False would have been better.)

### Sourcery Version

v0.10.2

### Code editor or IDE name and version

vscode

### OS name and version

Alpine 3.14.0.1 in WSL using VSCode on Windows

**Comment by Hellebore** [comment]
Thanks for raising!
I don't quite understand the difference in behaviour - could you give a few more details?

**Comment by ruancomelli** [comment]
@lonico we implemented a fix for this but. It will be available in the next nightly release and in v0.10.3.
Thanks for reporting this issue!

**Comment by Hellebore** [comment]
Now released in 0.10.3.
